google/ggrc-core
# [WIP] GGRC-6979 Disable Risk CRUD

# Issue description

Disable create/modify/delete of Risk for internal users.

# Steps to test the changes

1. Login to GGRC as external_app@example.com
2. Create risk. Should succeed.
3. Modify risk. Should succeed.
4. Delete risk. Should succeed.
5. Login to GGRC as global creator
6. Create risk. Should fail.
7. Modify risk. Should fail.
8. Delete risk. Should fail.

# Solution description

Change service resource for Risk model to ExternalResource which restrict requests from internal users.

# Sanity checklist




# PR Review checklist
